app/src/main/java/com/example/drawingactivity/DrawableAreaView.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.drawingactivity
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+
+/**
+ * This view is the drawable area of the app.
+ * drawing on this location will draw to a bitmap which is loaded
+ * this saves the paths that are drawn on it
+ */
+@SuppressLint("ClickableViewAccessibility")
+class DrawableAreaView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : View(context, attrs, defStyle) {
+
+    private var userDrawnBitmap: Bitmap? = null
+    private var intermediaryCanvas: Canvas? = null
+
+    var pathPaint = Paint().apply {
+        color = Color.BLACK
+        style = Paint.Style.STROKE
+        strokeWidth = 15f
```

Comment:
Constant

Reply:
I was adjusting the color of the pathPaint from the lambda in the Main activity, but I am going to add a variable instead since this gives it more access than is needed to the rest of the variables in the pathPaint.

added var paintColor = Color.BLACK and feed that into the paint instead

Reply:
This will default to Black in the event if of any issues with having a color set

Reply:
Not the color, for the stroke width

Reply:
Oh whoops , adding fixing now